web/app.js

```diff
@@ -699,7 +699,10 @@ const PDFViewerApplication = {
   },
 
   get supportsPrinting() {
-    return PDFPrintServiceFactory.instance.supportsPrinting;
+    return (
+      (typeof PDFJSDev === "undefined" || !PDFJSDev.test("GECKOVIEW")) &&
+      PDFPrintServiceFactory.instance.supportsPrinting
+    );
```

Comment:
It *just* occurred to me that these changes were not even necessary here :-)
The reason is you updated the `viewer.js` file to no longer bundle any printing code, hence the existing getter works fine given the defaults used in https://github.com/mozilla/pdf.js/blob/8a0ca04399cd1b4b4d1bb960685550e3f6ea4e03/web/app.js#L3143-L3151 and there being no code available to override it.